content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -315,7 +318,24 @@ Users of the REST API can sign CSRs by submitting an UPDATE request to the `stat
 subresource of the CSR to be signed.
 
 As part of this request, the `status.certificate` field should be set to contain the
-signed certificate.
+signed certificate. This field contains one or more PEM-encoded certificates.
+
+All PEM blocks must have the "CERTIFICATE" label, contain no headers,
+and the encoded data must be a BER-encoded ASN.1 Certificate structure
+as described in section 4 of RFC5280.
+
+Non-PEM content may appear before or after the CERTIFICATE PEM blocks and is unvalidated,
+to allow for explanatory text as described in section 5.2 of RFC7468.
```

Comment:
How about adding an example of valid data, prior to base64 encoding.
(the explanatory text could include a note that this whole block is what you base64 encode)?

Reply:
done